src/js/control-bar/text-track-controls/chapters-button.js

```diff
@@ -124,7 +98,8 @@ class ChaptersButton extends TextTrackButton {
 
         const mi = new ChaptersTrackMenuItem(this.player_, {
           cue,
-          track: chaptersTrack
+          track: chaptersTrack,
+          selectable: true
```

Comment:
They actually are supposed to get updated as mentioned in https://github.com/videojs/video.js/pull/3472 and https://github.com/videojs/video.js/issues/3447.